scanner_common.h

```diff
@@ -88,8 +86,4 @@ void a64_inline_hash_lookup(dbm_thread *thread_data, int basic_block, uint32_t *
                             uint32_t *read_address, enum reg rn, bool link, bool set_meta);
 #endif
 
-extern void inline_hash_lookup();
-extern void end_of_inline_hash_lookup();
-extern void inline_hash_lookup_get_addr();
-extern void inline_hash_lookup_data();
```

Comment:
It looks like these were some legacy functions from way back in the day. Well caught